eng/mgmt/automation/generate_utils.py

```diff
@@ -334,58 +338,66 @@ def generate_typespec_project(
         tspconfig_valid = True
         if url_match:
             # generate from remote url
-            cmd = ['npx', 'tsp-client', 'init', '--debug',
+            tsp_cmd = ['npx', 'tsp-client', 'init', '--debug',
                    '--tsp-config', tsp_project]
         else:
             # sdk automation
             tsp_dir = os.path.join(spec_root, tsp_project) if spec_root else tsp_project
             tspconfig_valid = validate_tspconfig(tsp_dir)
             repo = remove_prefix(repo_url, 'https://github.com/')
-            cmd = ['npx', 'tsp-client', 'init', '--debug',
+            tsp_cmd = ['npx', 'tsp-client', 'init', '--debug',
                    '--tsp-config', tsp_dir,
                    '--commit', head_sha,
                    '--repo', repo,
                    '--local-spec-repo', tsp_dir]
```

Comment:
check lint

Reply:
pylint doesn't complain

Reply:
I guess that more on the format area.

Maybe we can do a `black` on py someday.